### ISSUE TITLE
Switching source url doesn't behave correctly

### ISSUE THREAD

**Issue by htrandev172** [MelihAltintas/vue3-openlayers#386]
**Describe the bug**
https://stackblitz.com/edit/vue3-openlayers-template-q76erp?file=src/components/TheMap.vue,src/App.vue
Same behavior can be observed here: https://vue3openlayers.netlify.app/componentsguide/sources/xyz/

Please have a look at this demo using the latest version. When switching map url (using a computed property), it looks like the source does not clean up properly. If you click the Switch button on the demo, then try zoom in / out, you will see the previous map still flashing.

I suspect it has something to do with reusing the source on `useSource.ts` file in the recent version (I tested v10, v11). I switched to version 8 and this bug didn't happen.

**Affected version(s)**

Please run `npm list --depth=0 vue vue3-openlayers ol ol-ext ol-contextmenu` and paste the output below:

```
+-- ol-contextmenu@5.5.0
+-- ol-ext@4.0.23
+-- ol@9.2.4
+-- vue@3.5.11
`-- vue3-openlayers@11.2.1
```

Thanks. Please let me know if I can help with anything.

**Comment by htrandev172** [comment]
Sorry looks like this has been reported and fixed. Close.

**Comment by d-koppenhagen** [comment]
Issue should have been fixed with version 11.3.0
